quesma/schema/static_registry.go

```diff
@@ -39,3 +39,10 @@ func (e StaticRegistry) UpdateFieldEncodings(encodings map[FieldEncodingKey]Enco
 		e.FieldEncodings[k] = EncodedFieldName(v)
 	}
 }
+
```

Comment:
We should change all methods to be pointer receivers. These value receivers are tricky. Code looks "ok", but it don't work.

Reply:
You are right